doc/bips.md

```diff
@@ -1,4 +1,4 @@
-BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.18.0**):
+BIPs that are implemented by Dash Core (up-to-date up to **v18.0**):
```

Comment:
I don't think this is updated. @thephez you want to make sure this is updated?

Reply:
See #4780